cmd/operator-sdk/run/cmd.go

```diff
@@ -60,13 +62,10 @@ func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "run",
 		Short: "Run an Operator in a variety of environments",
-		Long: `This command will run or deploy your Operator in two different modes: locally
-and using OLM. These modes are controlled by setting --local and --olm run mode
-flags. Each run mode has a separate set of flags that configure 'run' for that
-mode. Run 'operator-sdk run --help' for more information on these flags.
-
-Read more about the --olm run mode and configuration options here:
-https://sdk.operatorframework.io/docs/olm-integration/cli-overview
+		Long: `This command has subcommands that will run or deploy your Operator in two
+different modes: locally and using OLM. These modes are controlled by using 'local'
+or 'packagemanifests' subcommands. Run 'operator-sdk run --help' for more
```

Comment:
Will we have the run bundle as well? 
It shows a little confuse? Was not the olm command able to work with bundles?

Reply:
It could theoretically work with bundles but is not recommended. Once `run bundle` is added we can update this description.

Reply:
cacher the idea .. tks for the clarification.

Reply:
I am ok with here in this PR. It is may out of scope. However, since I just checked these here then, I'd like to share the thought.  

We are working on to have: 

- operator-sdk run packagemanifests -> run olm project integration with pkg manifest config 
- operator-sdk run  bundle -> run olm project  integration with pkg manifest config 
- operator-sdk run local -> run the project locally

However, may not make more sense something as : 

- operator-sdk run olm [OPTIONS: --config='pkg' ] -> run olm project integration  where the default config is by using bundle 
- operator-sdk run local -> run the project locally

Reply:
`run bundle` and `run packagemanifests` will have different flag sets and underlying concept (bundle vs package manifests format). I think it makes sense to differentiate them by subcommand rather than run mode, and makes deprecation of package manifests easier down the line (although we have no concrete plan to deprecate).

Reply:
Fair enough